packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -260,8 +285,6 @@ class AnalyzeCommand extends FlutterCommand {
 
     int exitCode = await process.exitCode;
 
-    host.deleteSync(recursive: true);
```

Comment:
It's the unused import for the `lib/flutter_sprites.dart` reference. I'll move the dir delete back and filter out analysis errors from the generated entry-point file.